src/allmydata/util/configutil.py

```diff
@@ -42,11 +42,12 @@ def get_config(tahoe_cfg):
     """
     config = SafeConfigParser()
     with open(tahoe_cfg, "r") as f:
-        # On Python 2, where we read in bytes, skip any initial Byte Order
-        # Mark. Since this is an ordinary file, we don't need to handle
-        # incomplete reads, and can assume seekability.
+        # Skip any initial Byte Order Mark. Since this is an ordinary file, we
+        # don't need to handle incomplete reads, and can assume seekability.
         if PY2 and f.read(3) != b'\xEF\xBB\xBF':
             f.seek(0)
+        if PY3 and f.read(1) != u"\uFEFF":
```

Comment:
So sad...  Maybe I'll go switch us from ConfigParser to configobj.  The latter seems better and lets us have unicode on both Python versions, it seems.

Reply:
There's also https://pypi.org/project/configparser/, which is a backport so less intrusive. Maybe worth a follow-up PR? Or perhaps put this on hold and do it first?

Reply:
Went down this route.